docs/topics/wasm/wasm-js-interop.md

```diff
@@ -296,17 +296,34 @@ You can use nullable versions of these types as well.
 
 JavaScript values are represented in Kotlin using the `JsAny` type and its subtypes.
 
-The standard library provides representation for some of these types:
+The Kotlin/Wasm standard library provides representation for some of these types:
 * Package `kotlin.js`:
     * `JsAny`
     * `JsBoolean`, `JsNumber`, `JsString`
     * `JsArray`
     * `Promise`
+
+The [`kotlinx-browser` library](https://github.com/kotlin/kotlinx-browser), a stand-alone
+library including web-related APIs, also provides
+representation for some of the `JsAny` type and its subtypes:
```

Comment:
Maybe better to say:
`
... a stand-alone library which provides JavaScript browser APIs:
`

Reply:
or
`... a stand-alone library which provides web-related browser APIs:`

Reply:
I do not think than we need `also offers
representation for some of the `JsAny` type and its subtypes:`